### PR TITLE
change misleading description of m_mul

### PR DESCRIPTION
**Pull request by mikael-epigram**
[CosineDecayRestarts](https://www.tensorflow.org/api_docs/python/tf/keras/optimizers/schedules/CosineDecayRestarts) has several parameters in addition to the inital learning rate, all of which are described in the paragraph: 

> The learning rate multiplier first decays from 1 to `alpha` for `first_decay_steps` steps. Then, a warm restart is performed. Each new warm restart runs for `t_mul` times more steps and with `m_mul` times **smaller** initial learning rate.

(my emphasis)

`m_mul` is the inverse of what is being described here. It should therefore be `... m_mul times larger initial learning rate`.

The args section does little to explain the effect of an increase or decrease in the parameter:

`m_mul`
> A scalar float32 or float64 Tensor or a Python number. Used to derive the initial learning rate of the i-th period:

### CODE REVIEW

**Comment by gbaned** [comment]
@mikael-epigram  Can you please sign CLA. Thanks!

File: keras/optimizer_v2/learning_rate_schedule.py [hunk 1]
```diff
@@ -662,7 +662,7 @@ class CosineDecayRestarts(LearningRateSchedule):
   The learning rate multiplier first decays
   from 1 to `alpha` for `first_decay_steps` steps. Then, a warm
   restart is performed. Each new warm restart runs for `t_mul` times more
-  steps and with `m_mul` times smaller initial learning rate.
+  steps and with `m_mul` times larger initial learning rate.
```
Comment (qlzh727) [inline keras/optimizer_v2/learning_rate_schedule.py:665]: I think the smaller makes more correct impression here (this is a Decay, not a learning rate amplifier). Everytime when a warm start happens, the new learning rate will be existing learning rate times m_mul. Did I miss anything?

You can also take a look for https://towardsdatascience.com/https-medium-com-reina-wang-tw-stochastic-gradient-descent-with-restarts-5f511975163

Comment (mikael-epigram) [reply keras/optimizer_v2/learning_rate_schedule.py:665]: You are right in that you would typically want to have smaller learning rates on restarts. But mathematically, `m_mul` represents the inverse.

If you set `m_mul=2` you get a 2 times larger initial learning rate. 

If you set `m_mul=0.5` you get a 0.5 times larger (= 2 times smaller) initial learning rate.

Comment (qlzh727) [reply keras/optimizer_v2/learning_rate_schedule.py:665]: Sure. How about just remove the smaller/larger here since it is really depending the m_mul value here. "`m_mul` times initial learning rate" as the new learning rate seems reasonable to me as well.

Comment (mikael-epigram) [reply keras/optimizer_v2/learning_rate_schedule.py:665]: I agree, that removes the ambiguity quite nicely.

**Comment by qlzh727** [comment]
Thanks for the fix.

**Review by qlzh727 — APPROVED** [review]
